deeposlandia/utils.py

```diff
@@ -62,6 +62,29 @@ def label_building(filtered_image, label_ids, dataset='mapillary'):
         available_labels[available_labels==255] = 1
     return {i: 1 if i in available_labels else 0 for i in label_ids}
 
+def geojson_label_building(items, label_names):
+    """Build a dict that summarizes image label composition
+
+    Parameters
+    ----------
+    items : pandas.Series
+        Type of items that must be summarized
+    label : list
+        Labels contained into the reference classification
+
+    Returns
+    -------
+    dict
+        Label ids which occur (or not) in the image
+    """
+    tile_labels = [s.lower() for s in items.unique().tolist()]
```

Comment:
maybe you can do this `items.unique().str.lower().tolist()`

Reply:
The `geojson_label_building` is even useless, I removed it. This particular point is then solved.